yarn.lock

```diff
@@ -3765,20 +3765,13 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@tsd/typescript@npm:^4.9.5, @tsd/typescript@npm:~4.9.5":
+"@tsd/typescript@npm:^4.9.5":
   version: 4.9.5
   resolution: "@tsd/typescript@npm:4.9.5"
   checksum: f9f01ecf2094e0ac83e56cc57430bdf81da82aa20424280002b44ea29cdc171388a9777262257f550800bf76567774521728e1ef6e191633ae6f723ee64d396a
   languageName: node
   linkType: hard
 
-"@tsd/typescript@npm:~5.0.2":
```

Comment:
For example, this is a copy of the whole `typescript` package, which you don’t use. It was here just because `tsd` ships it.